src/integration-tests/introspector/checkMIIBeans.inputt

```diff
@@ -0,0 +1,27 @@
+/Log/base_domain,FileName,logs/base_domain.log,/shared/logs/base_domain.log
```

Comment:
is this file able to have a copyright notice added?

Reply:
This one is  not same as the checkBeans.inputt

Reply:
sorry, i don't understand, can it have a copyright notice added please? if not, why not?

Reply:
This data file is a copy of the original one (also without copyrights) but looking at the checkBeans.py actually it can take comments, so I am adding copyrights to both